setup/pattoo_installation.py

```diff
@@ -380,10 +379,11 @@ def main():
             # Import db after pip3 packages are installed
             from _pattoo import db
             db.install()
-            systemd.install(daemon_list=daemon_list,
-                            template_dir=template_dir,
-                            installation_dir=installation_dir,
-                            verbose=args.verbose)
+            if getpass.getuser() == 'root':
+                systemd.install(daemon_list=daemon_list,
+                                template_dir=template_dir,
```

Comment:
Do you think we should limit database installation to `PATTOO_CONFIGDIR` environments where the path includes the word `unittest`?

Reply:
This would help with some of the earlier issues we had where too much flexibility got us into problems.

Reply:
I think that it would be best to.
Currently, if the user doesn't set the `PATTOO_CONFIGDIR`, the installation will automatically stop since the normal user won't have permissions for `/etc/pattoo`.

Reply:
Ok, then implement it that way. `pattoo` user, unittest `PATTOO_CONFIGDIR` or otherwise exit with message